### PR TITLE
Functions, physics, pyglet, sets and strings translated into English

### PR DESCRIPTION
**Pull request by muzikovam**

### CODE REVIEW

**Comment by encukou** [comment]
We already have one English cheatsheet, and it uses `-en` (not `-eng`) for the language code.
Is `-eng` on purpose? Could you rename the files to `-en`?

File: game-physics/game-physics-eng.svg [hunk 1]
```diff
@@ -0,0 +1,1734 @@
...
+      </g>
+      <g
+         transform="matrix(1.2016036,0,0,1.2016036,-845.44481,34.087128)"
+         id="g11217"
+         style="stroke:#ffffff;stroke-opacity:1">
+        <path
+           inkscape:connector-curvature="0"
+           style="fill:#000000;fill-opacity:1;fill-rule:evenodd;stroke:#ffffff;stroke-width:3;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-opacity:1"
+           d="m 955.38921,118.27679 c -8.74072,-25.62252 84.29659,1.20001 65.33669,16.35792 16.2044,-15.19065 53.4145,53.76615 34.3022,52.35599 12.9176,1.72964 16.1984,40.62825 -1.8054,32.49683 17.6813,-1.98267 9.1505,46.91096 -5.4161,46.93985 16.3399,6.50189 -5.5361,44.25827 -23.4699,41.52372 3.839,8.27988 -8.0364,32.07079 -21.6641,28.88606 11.3718,6.54314 -21.48749,28.45123 -29.26327,22.34726 1.38396,5.48873 -13.53684,7.39344 -18.02012,1.57401 -4.48328,4.91674 -21.54109,7.55421 -18.02012,-1.57401 -5.51906,7.458 -42.68229,-17.15815 -30.8592,-22.34726 -13.62823,3.18473 -26.46104,-19.96788 -22.622,-28.24776 -17.93387,2.73455 -35.29648,-32.23633 -23.46992,-42.80032 -14.56659,-0.0289 -21.47904,-44.67346 -6.05444,-46.30155 -18.49331,10.83227 -15.5054,-32.49683 -2.44368,-
```
Comment (encukou) [inline game-physics/game-physics-eng.svg]: Typo: `Rectanfles` → `Rectangles`

File: game-physics/game-physics-eng.svg [hunk 2]
```diff
@@ -0,0 +1,1734 @@
...
+         y="566.92908"
+         x="24.80315"
+         sodipodi:role="line"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         id="tspan7749">Turtle moves a distance d in direction α.</tspan><tspan
+         y="581.10406"
+         x="24.80315"
+         sodipodi:role="line"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         id="tspan632">How to convert it into x, y coordinates (and back?</tspan><tspan
```
Comment (encukou) [inline game-physics/game-physics-eng.svg]: Missing closing parenthesis: `How to convert it into x, y coordinates (and back?`

File: pyglet/pyglet-basics-eng.svg [hunk 3]
```diff
@@ -0,0 +1,1057 @@
...
+       id="text7248"
+       y="386.22046"
+       x="14.173229"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;line-height:0%;font-family:'Bree Serif';-inkscape-font-specification:'Bree Serif';text-align:start;letter-spacing:0px;word-spacing:0px;text-anchor:start;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+       xml:space="preserve"><tspan
+         y="386.22046"
+         x="14.173229"
+         sodipodi:role="line"
+         id="tspan4704"
+         style="font-size:11.25px;line-height:1.25">Iinstallation (from command line)</tspan></text>
```
Comment (encukou) [inline pyglet/pyglet-basics-eng.svg]: Typo: `Iinstallation` → `Installation` (extra i)

File: pyglet/pyglet-basics-eng.svg [hunk 4]
```diff
@@ -0,0 +1,1057 @@
...
+       xml:space="preserve"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;line-height:0%;font-family:'Bree Serif';-inkscape-font-specification:'Bree Serif';text-align:start;letter-spacing:0px;word-spacing:0px;text-anchor:start;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+       x="14.173229"
+       y="499.60629"
+       id="text4776"><tspan
+         id="tspan4778"
+         sodipodi:role="line"
+         x="14.173229"
+         y="499.60629"
+         style="font-size:11.25px;line-height:1.25">Interesting eventsi</tspan></text>
```
Comment (encukou) [inline pyglet/pyglet-basics-eng.svg]: Typo: `eventsi` → `events`

File: sets/sets-eng.svg [hunk 5]
```diff
@@ -0,0 +1,1466 @@
...
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         sodipodi:role="line"
+         x="202.18823"
+         y="654.19611"
+         id="tspan5590"> </tspan><tspan
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         sodipodi:role="line"
+         x="202.18823"
+         y="668.37109"
+         id="tspan5592">True if sets are disjoint? (tj. they don't have any</tspan><tspan
```
Comment (encukou) [inline sets/sets-eng.svg]: `tj.` is Czech, use `e.g.`

File: strings/strings-eng.svg [hunk 6]
```diff
@@ -0,0 +1,1560 @@
...
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         sodipodi:role="line"
+         x="123.51142"
+         y="570.89062"
+         id="tspan4406">1st appearance from position n</tspan><tspan
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         sodipodi:role="line"
+         x="123.51142"
+         y="585.06561"
+         id="tspan4408">Removes whitespaces* - beginning </tspan><tspan
```
Comment (encukou) [inline strings/strings-eng.svg]: “whitespace” (singular) is one or more spaces/newlines. No need to use the plural.

File: sets/sets-eng.svg (hunk 5 above)
Comment (hroncok) [reply sets/sets-eng.svg]: use [i.e.](https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/)

**Pull request merged**